### PR TITLE
Add 24 descriptions (1 of 6)

### PR DESCRIPTION
**Pull request by rasa**

### CODE REVIEW

File: bucket/faas-cli.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://www.openfaas.com/",
+    "description": "OpenFaaS (Functions as a Service) is a framework for building serverless functions with Docker and Kubernetes which has first class support for metrics. (CLI)",
```
Comment (niheaven) [inline bucket/faas-cli.json]:
```suggestion
    "description": "A framework for building serverless functions with Docker and Kubernetes which has first class support for metrics. (CLI)",
```

File: bucket/gdb.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "http://tdm-gcc.tdragon.net/",
+    "description": "GDB, the GNU Project debugger, allows you to see what is going on `inside' another program while it executes -- or what another program was doing at the moment it crashed. (TDM port)",
```
Comment (niheaven) [inline bucket/gdb.json]:
```suggestion
    "description": "GNU Project debugger that allows you to see what is going on `inside' another program while it executes -- or what another program was doing at the moment it crashed. (TDM port)",
```

Comment (Ash258) [reply bucket/gdb.json]: Small change for @niheaven suggestion to keep same quote

```suggestion
    "description": "GNU Project debugger that allows to see what is going on 'inside' another program while it executes -- or what another program was doing at the moment it crashed. (TDM port)"
```

File: bucket/gcc.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://mingw-w64.org",
+    "description": "GNU Compiler Collection (GCC) includes front ends for C, C++, Objective-C, Fortran, Ada, Go, and D, as well as libraries for these languages (libstdc++,...). (MingW port)",
```
Comment (Ash258) [inline bucket/gcc.json]:
```suggestion
    "description": "GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, Go, and D, as well as libraries for these languages (libstdc++,...). (MingW port)",
```

File: bucket/fnproject.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://fnproject.io/",
+    "description": "An open-source container-native serverless platform that you can run anywhere -- any cloud or on-premise.",
```
Comment (Ash258) [inline bucket/fnproject.json]: I personally do not like adress user directly in description. Rather have neutral description.

```suggestion
    "description": "A container-native serverless platform that can be runned anywhere (any cloud or on-premise).",
```

File: bucket/flyway.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://flywaydb.org/",
+    "description": "An open-source database migration tool, that favors simplicity and convention over configuration.",
```
Comment (Ash258) [inline bucket/flyway.json]:
```suggestion
    "description": "Database migration tool that favors simplicity and convention over configuration.",
```

File: bucket/flow.json [hunk 6]
```diff
@@ -1,5 +1,6 @@
 {
     "version": "0.105.2",
+    "description": "A static type checker for your JavaScript code.",
```
Comment (Ash258) [inline bucket/flow.json]:
```suggestion
    "description": "A static type checker for JavaScript code.",
```

File: bucket/flatc.json [hunk 7]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://google.github.io/flatbuffers/index.html",
+    "description": "An efficient cross-platform serialization library for C++, C#, C, Go, Java, JavaScript, Lobster, Lua, TypeScript, PHP, Python, and Rust.",
```
Comment (Ash258) [inline bucket/flatc.json]: No need to keep all languages.

```suggestion
    "description": "An efficient cross-platform serialization library for multiple languages.",
```

File: bucket/figlet.json [hunk 8]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://github.com/lukesampson/figlet",
+    "description": "A program that makes large letters out of ordinary text.",
```
Comment (Ash258) [inline bucket/figlet.json]:
```suggestion
    "description": "Create large letters using ASCII art out of ordinary text.",
```

File: bucket/far.json [hunk 9]
```diff
@@ -4,6 +4,7 @@
         "url": "https://farmanager.com/license.php"
     },
     "homepage": "https://farmanager.com/",
+    "description": "A text-mode program for managing files and archives.",
```
Comment (Ash258) [inline bucket/far.json]:
```suggestion
    "description": "A text-mode files and archives manager.",
```

File: bucket/doxygen.json [hunk 10]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "http://www.doxygen.nl/",
+    "description": "A documentation generator, a tool for writing software reference documentation.",
```
Comment (Ash258) [inline bucket/doxygen.json]:
```suggestion
    "description": "Documentation generator from annoted source code.",
```

File: bucket/draft.json [hunk 11]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://github.com/Azure/draft",
+    "description": "A tool for developers to create cloud-native applications on Kubernetes.",
```
Comment (Ash258) [inline bucket/draft.json]:
```suggestion
    "description": "Create cloud-native applications on Kubernetes.",
```

**Pull request merged**